lib/va_profile/v2/contact_information/service.rb

```diff
@@ -34,14 +34,16 @@ class Service < VAProfile::Service
         # @return [VAProfile::V2::ContactInformation::PersonResponse] wrapper around an person object
         def get_person
           with_monitoring do
-            raw_response = perform(:get, "#{MPI::Constants::VA_ROOT_OID}/#{ERB::Util.url_encode(icn_with_aaid)}")
+            verify_user!
+
+            raw_response = perform(:get, "#{MPI::Constants::VA_ROOT_OID}/#{ERB::Util.url_encode(vaprofile_aaid)}")
             PersonResponse.from(raw_response)
           end
         rescue Common::Client::Errors::ClientError => e
           if e.status == 404
             log_exception_to_sentry(
               e,
-              { vet360_id: },
+              { vet360_id: @user&.vet360_id },
               { va_profile: :person_not_found },
```

Comment:
The `vet360_id` method was unnecessary after this refactor so `@user&.vet360_id` has to be defined.